File: 백준/Bronze/2566. 최댓값/최댓값.java

import java.util.Scanner;

public class Main {
    public static void main(String[] args) {

        Scanner sc = new Scanner(System.in);

        int[][] numArray = new int[9][9];
        int maxNum = 0;
        int maxNumRow = 0;
        int maxNumCol = 0;

        for (int i = 0; i < 9; i++) {
            for (int j = 0; j < 9; j++) {
                numArray[i][j] = sc.nextInt();
                if (numArray[i][j] >= maxNum) {
                    maxNum = numArray[i][j];
                    maxNumRow = i + 1;
                    maxNumCol = j + 1;
                }
            }
        }
        System.out.println(maxNum);
        System.out.println(maxNumRow + " " + maxNumCol);
    }
}

File: 백준/Bronze/2566. 최댓값/최댓값.cc
#include <iostream>

using namespace std;

int main() {
	
	int arr[9][9];
	int maxNum = 0;
	int maxNumRow = 0;
	int maxNumCol = 0;

	for (int i = 0; i < 9; i++)
	{
		for (int j = 0; j < 9; j++)
		{
			cin >> arr[i][j];
			if (arr[i][j] >= maxNum)
			{
				maxNum = arr[i][j];
				maxNumRow = i;
				maxNumCol = j;
			}
		}
	}
	cout << maxNum << endl;
	cout << maxNumRow + 1 << " " << maxNumCol + 1 << endl;
}